src/sentry/rules/history/backends/postgres.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from sentry.models import RuleFireHistory
+from sentry.rules.history.base import RuleHistoryBackend
+
+if TYPE_CHECKING:
```

Comment:
wat

Reply:
lol, just loads things when the type checker is running. Helps for circular references in some cases